src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -2045,11 +2045,18 @@ WaterBreath.Timer = 180000
 EnableLowLevelRegenBoost = 1
 
 #
-#    Rate.MoveSpeed
-#        Description: Movement speed rate.
+#    Rate.MoveSpeed.Player
```

Comment:
I would personally prefer `Rate.MoveSpeed.PC`/`Rate.MoveSpeed.NPC` myself just for consistency with e.g. unit flag descriptions.

Reply:
This change was done based on this request:
https://github.com/azerothcore/azerothcore-wotlk/pull/18623#discussion_r1590175768